content/events-calendar/2024/Unlocking-the-Full-Potential-The-Harmonious-Dance-of-EF-Core-and-SQL-Server.json

```diff
@@ -0,0 +1,45 @@
+{
+  "title": "Unlocking the Full Potential: The Harmonious Dance of EF Core and SQL Server",
+  "url": "https://www.ssw.com.au/netug/brisbane",
+  "thumbnail": "/images/thumbs/thumb-brisbaneUG.jpg",
+  "thumbnailDescription": "Brisbane Full Stack User Group",
+  "presenterName": "Jernej Kavka",
+  "presenterProfileUrl": "http://ssw.com.au/people/jernej-kavka",
```

Comment:
Confused why we have these two fields when we also have jernej-kavka.mdx

~I would expect this to be a reference type to the presenter file instead~

Line 39 has the presenters list... I think we can remove these fields and use that instead

Reply:
There are cases where we have presenterName but nothing in presentersList. 
This might suit the case when marketing does not want to add a new presenter to presenters collection (i.e. a one-off)